### PR TITLE
Fix/students table

### PR DESCRIPTION
**Pull request by nadiakohli**

### CODE REVIEW

File: front-end/src/components/common/SubscriptionsTable/index.jsx [hunk 1]
```diff
@@ -38,12 +41,25 @@ const TeacherTable = ({ subscriptions, deleteSubscriptionById }) => {
...
+        >
           <TableRow>
-            <TableCell>Avatar</TableCell>
-            <TableCell align='left'>Full Name</TableCell>
-            <TableCell align='left'>Email</TableCell>
-            <TableCell align='left'>Following</TableCell>
+            <TableCell sx={{ maxWidth: 65 }} align='center'>
+              Avatar
+            </TableCell>
+            <TableCell align='center'>Full Name</TableCell>
```
Comment (6eluuNiGer) [inline front-end/src/components/common/SubscriptionsTable/index.jsx]: Better create const for tableCell and use method map

File: front-end/src/pages/StudentSubscriptions/index.jsx [hunk 2]
```diff
@@ -40,18 +43,26 @@ const StudentSubscriptions = () => {
       return remove;
     } catch (e) {
       console.log(e);
```
Comment (6eluuNiGer) [inline front-end/src/pages/StudentSubscriptions/index.jsx]: remove log

File: front-end/src/pages/TeacherSubscriptions/index.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  };
+
+  const deleteSubscriptionById = async (id) => {
+    try {
+      setIsLoading(true);
+      const necessarySubscription = subscriptions.find((subscription) => {
+        return subscription._id === id;
+      });
+      const remove = await deleteSubscription(necessarySubscription._id);
```
Comment (6eluuNiGer) [inline front-end/src/pages/TeacherSubscriptions/index.jsx:40]: const remove, it wrong 
Please, change name

File: front-end/src/components/common/SubscriptionsTable/index.jsx (hunk 1 above)
Comment (nadiakohli) [reply front-end/src/components/common/SubscriptionsTable/index.jsx]: fix it

File: front-end/src/pages/TeacherSubscriptions/index.jsx (hunk 3 above)
Comment (nadiakohli) [reply front-end/src/pages/TeacherSubscriptions/index.jsx:40]: fix it

**Review by 6eluuNiGer — APPROVED** [review]

**Pull request merged**